eng/pipelines/common/push-nightly.yml

```diff
@@ -55,7 +55,7 @@ stages:
           - task: DownloadPipelineArtifact@2
             inputs:
               artifactName: vs-msi-nugets
-              downloadPath: $(Build.StagingDirectory)\nuget-signed
+              downloadPath: $(Build.StagingDirectory)\PackageArtifacts
```

Comment:
It may be worth removing the nuget-signed artifact download above, or adding a filter to this to reduce duplicates:
```yaml
  patterns: |
    **/*msi*.nupkg
```

Or a third option would be to download from the drop services:
```yaml
- task: ms-vscs-artifact.build-tasks.artifactDropDownloadTask-1.artifactDropDownloadTask@1
  displayName: Download path/to/drop/nugets
  inputs:
    dropServiceURI: https://devdiv.artifacts.visualstudio.com/DefaultCollection
    buildNumber: path/to/drop/nugets
    destinationPath: $(Build.StagingDirectory)\nuget-signed
```

Reply:
Thats done by the yaml templates , I don't see the vs-msi-nugets folder anymore,

Reply:
We actually don t need the msg